### DISCUSSION TITLE
Issue thread breaks on Github iOS

### DISCUSSION THREAD

**Discussion by aaanh** [community/community#37740] [Mobile]
When viewing an issue discussion on the Github iOS app, the thread layout breaks, only shows the first reply to the issue and the rest of the replies are separated by a huge blank scroll to get to the bottom.

Please refer to the screenshots for a better idea of the bug.

Screenshot 1 denotes the end of the 1st comment.
![image](https://user-images.githubusercontent.com/37283437/198853243-7d4a7f64-0ef5-4896-956e-9ea57410c2ef.png)

Screenshot 2 denotes the huge blank vertical scroll.
![image](https://user-images.githubusercontent.com/37283437/198853237-13d378cd-a23a-4f3e-a738-fcca79f1b01d.png)

Step(s) to reproduce:
- Open to view any issue for the first time.
- Refreshing the issue page by pull down the page at the top did not resolve the bug
- Restarting did not resolve the bug
- BUT, viewing another issue and going back somehow "resets" the layout and the issue thread is displayed correctly.

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.86.0 (81658795)
Device: iPhone 13 (iOS 16.1)
</details>

**Comment by salemahoo** [comment]
> When viewing an issue discussion on the Github iOS app, the thread layout breaks, only shows the first reply to the issue and the rest of the replies are separated by a huge blank scroll to get to the bottom.
> 
> Please refer to the screenshots for a better idea of the bug.
> 
> Screenshot 1 denotes the end of the 1st comment.
> ![image](https://user-images.githubusercontent.com/37283437/198853243-7d4a7f64-0ef5-4896-956e-9ea57410c2ef.png)
> 
> Screenshot 2 denotes the huge blank vertical scroll.
> ![image](https://user-images.githubusercontent.com/37283437/198853237-13d378cd-a23a-4f3e-a738-fcca79f1b01d.png)
> 
> Step(s) to reproduce:
> - Open to view any issue for the first time.
> - Refreshing the issue page by pull down the page at the top did not resolve the bug
> - Restarting did not resolve the bug
> - BUT, viewing another issue and going back somehow "resets" the layout and the issue thread is displayed correctly.
> 
> <details>
> <summary>ℹ️ Additional Debug Info</summary>
> GitHub for iOS, Version 1.86.0 (81658795)
> Device: iPhone 13 (iOS 16.1)
> </details>

**Comment by radar-blip** [comment]
I believe you need to file a bug report for this in order for the team to be made aware of it, and fix it. 
You can do so, here: [Bug-Report](https://support.github.com/contact/bug-report)

**Reply by aaanh** [reply]
Oh, okay, thanks for letting me know. I used the mobile app to create this discussion but I now know that it's strictly community discussions. 😅

I'll go ahead and submit the bug report using the link you gave.

Thanks again 😇

**Reply by radar-blip** [reply]
You're welcome. Hope it gets fixed! 😄
